scenes/game/game.tscn

```diff
@@ -1,13 +1,25 @@
-[gd_scene load_steps=5 format=3 uid="uid://dkufj4sv64avp"]
+[gd_scene load_steps=6 format=3 uid="uid://dkufj4sv64avp"]
 
 [ext_resource type="Script" uid="uid://d37a6aqva6idi" path="res://scenes/game/game.gd" id="1_b8j6q"]
 [ext_resource type="PackedScene" uid="uid://cvcpo1lcpdoyt" path="res://scenes/level_ui/player_level_ui.tscn" id="2_gtdlk"]
 [ext_resource type="PackedScene" uid="uid://g3e0eodhl7xl" path="res://menus/menu.tscn" id="3_gr4f4"]
 [ext_resource type="PackedScene" uid="uid://coxcqmcg3kk6h" path="res://scenes/level_ui/tree_level_ui.tscn" id="3_wclvw"]
 
+[sub_resource type="Environment" id="Environment_lfrn8"]
+background_energy_multiplier = 3.08
```

Comment:
Please no not do this haha
Add a [DirectionalLight3D ](https://docs.godotengine.org/en/stable/classes/class_directionallight3d.html)to the scene